app/code/Magento/Shipping/Model/Shipping/LabelGenerator.php

```diff
@@ -169,24 +173,32 @@ public function createPdfPageFromImageString($imageString)
             DirectoryList::TMP
         );
         $directory->create();
-        $image = @imagecreatefromstring($imageString);
-        if (!$image) {
+
+        try {
+            // phpcs:ignore Magento2.Functions.DiscouragedFunction
+            $image = imagecreatefromstring($imageString);
```

Comment:
The `@` should be preserved for 7.4. Even in 8.0 doesn't silence the fatal errors, however we should keep it there. Please share your opinion on that.